src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -45,8 +45,21 @@ public final class FieldValidator {
     public static final String SECTION_NAME_FIELD_NAME = "section name";
     public static final int SECTION_NAME_MAX_LENGTH = 60;
 
+    public static final String ACCOUNT_REQUEST_INSTITUTE_NAME_FIELD_NAME = "university/school/institute name";
+    public static final int ACCOUNT_REQUEST_INSTITUTE_NAME_MAX_LENGTH = 60;
+
+    public static final String ACCOUNT_REQUEST_COUNTRY_NAME_FIELD_NAME = "country name";
+    public static final int ACCOUNT_REQUEST_COUNTRY_NAME_MAX_LENGTH = 60;
+
     public static final String INSTITUTE_NAME_FIELD_NAME = "institute name";
-    public static final int INSTITUTE_NAME_MAX_LENGTH = 64;
+    public static final int INSTITUTE_NAME_MAX_LENGTH =
+            ACCOUNT_REQUEST_INSTITUTE_NAME_MAX_LENGTH + ACCOUNT_REQUEST_COUNTRY_NAME_MAX_LENGTH + 8;
+
+    public static final String ACCOUNT_REQUEST_HOME_PAGE_URL_FIELD_NAME = "url";
+    public static final int ACCOUNT_REQUEST_HOME_PAGE_URL_MAX_LENGTH = 500;
+
+    public static final String ACCOUNT_REQUEST_COMMENTS_FIELD_NAME = "comment/query";
```

Comment:
Which one are you referring to? If you are referring to ACCOUNT_REQUEST_COMMENTS_FIELD_NAME, I don't see any reason to name it comments. If you are referring to "comment/query", they are not chosen randomly but considering what the generated error message will be, because that's what will be shown to users.